src/dialogflow_ros/dialogflow_node.py

```diff
@@ -11,19 +7,27 @@
 # Authentication
 SESSION_ID = ''
 
-class ApiaiNode(object):
+
+class DialogflowNode(object):
+    """
+    ROS node for the Dialogflow natural language understanding.
+
+    Dialogflow (previously API.ai) parses natural language into a json string containing the semantics of the text. This
+    nose wraps that into a ROS actionlib interface.
+    """
     def __init__(self):
-        rospy.init_node('apiai_node')
+        rospy.init_node('dialogflow_node')
         rospy.Subscriber("speech", String, self.speech_callback, queue_size=10)
         try:
-            CLIENT_ACCESS_TOKEN = rospy.get_param("~client_access_token")
-        except Exception as e:
+            self._client_access_token = rospy.get_param("~client_access_token")
+        except rospy.ROSException:
             rospy.logfatal("Missing required ROS parameter client_access_token")
             exit(1)
 
-        self.ai = apiai.ApiAI(CLIENT_ACCESS_TOKEN)
+        self.ai = apiai.ApiAI(self._client_access_token)
```

Comment:
api dit not change?

Reply:
why is it still apiai?

Reply:
I guess they didn't want to break the API, so this is remains unchanged for now.